src/main/java/org/vaadin/example/bookstore/ui/inventory/ProductForm.java

```diff
@@ -99,18 +103,18 @@ public ProductForm(InventoryViewLogic sampleCrudLogic) {
 
         viewLogic = sampleCrudLogic;
 
-        productName = new TextField("Product name");
+        productName = new TextField(resourceBundle.getString("product_name"));
         productName.setWidth("100%");
         productName.setRequired(true);
         productName.setValueChangeMode(ValueChangeMode.EAGER);
         content.add(productName);
 
-        price = new TextField("Price");
+        price = new TextField(resourceBundle.getString("price"));
         price.setSuffixComponent(new Span("€"));
         price.addThemeVariants(TextFieldVariant.LUMO_ALIGN_RIGHT);
```

Comment:
All number field alignments should be left.
In the enum file TextFieldVariant, should have LUMO_ALIGN_LEFT.

Reply:
Updated.